src/tests/account/test_account.cairo

```diff
@@ -100,9 +108,11 @@ fn deploy_erc20(recipient: ContractAddress, initial_supply: u256) -> IERC20Dispa
 fn test_constructor() {
     let mut state = STATE();
     Account::constructor(ref state, PUBLIC_KEY);
-    assert(
-        Account::PublicKeyImpl::get_public_key(@state) == PUBLIC_KEY, 'Should return public key'
-    );
+
+    let event = testing::pop_log::<OwnerAdded>(ZERO()).unwrap();
+    assert(event.new_owner_guid == PUBLIC_KEY, 'Invalid owner key');
```

Comment:
let's incorporate the utils and extra checks from [the Ownable PR ](https://github.com/OpenZeppelin/cairo-contracts/pull/675/)

Reply:
why are we passing `ZERO()`?

Reply:
> why are we passing ZERO()?

`pop_log` needs a contract address to know where to listen for events (ref [here](https://github.com/starkware-libs/cairo/blob/c3388d965be0ad3bd3ed30a749e98ba92e080f52/corelib/src/starknet/testing.cairo#L80)). Since we're neither deploying the contract nor setting the contract address in this particular test, its contract address is zero. It's different in [this case](https://github.com/andrew-fleming/cairo-contracts/blob/add-account-events/src/tests/account/test_account.cairo#L447-L461), for example, because we're setting the contract address prior to the target call